python/reactivedataflow/reactivedataflow/build_execution_graph.py

```diff
@@ -176,7 +182,7 @@ def bind_inputs(
             node = nodes[nid]
             if isinstance(node, InputNode):
```

Comment:
you could replace the ifs with structural pattern matching, you can do:

```py
match(node):
    case InputNode():
        node.attach(inputs[nid])
    case VerbNode():
        ...
    case _:
        pass
```

Reply:
done!